app/src/main/java/at/bitfire/icsdroid/model/CredentialsModel.kt

```diff
@@ -5,9 +5,11 @@ import androidx.compose.runtime.mutableStateOf
 import androidx.compose.runtime.setValue
 import androidx.lifecycle.ViewModel
 import at.bitfire.icsdroid.db.entity.Credential
-import kotlinx.coroutines.flow.MutableStateFlow
+import dagger.hilt.android.lifecycle.HiltViewModel
+import javax.inject.Inject
 
-class CredentialsModel : ViewModel() {
+@HiltViewModel
+class CredentialsModel @Inject constructor(): ViewModel() {
```

Comment:
I don't know about this one, but is it necessary to always annotate as `HiltViewModel`and inject? Even though we are not using hilt here.

Reply:
Yes, unfortunately we need to do that, if we want to inject the viewmodel in other places.

The goal with introducing DI is to make the shift from ICSx5 to DAVx5 easier, so it makes sense to do everything the same way DAVx5 does.